frontend/src/components/projectListView/ProjectListView.tsx

```diff
@@ -0,0 +1,252 @@
+import { Task } from '../../pages/projectView/ProjectView';
+import { useState, useEffect } from 'react';
+import ListTable from '../listTable/ListTable';
+import { AddCircleOutline } from '@mui/icons-material';
+import { TextField } from '@mui/material';
+import './projectList.css';
+import { v4 as uuid } from 'uuid';
+import { useParams } from 'react-router-dom';
+import { User, getAuth, onAuthStateChanged } from '@firebase/auth';
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+
+const ProjectListView = ({ tasks }: { tasks: Task[] }) => {
+  const [allTasks, setAllTasks] = useState<Task[]>(tasks);
+
+  const [documentation, setDocumentation] = useState<Task[]>([]);
+  const [ongoing, setOngoing] = useState<Task[]>([]);
+  const [todo, setTodo] = useState<Task[]>([]);
+  const [done, setDone] = useState<Task[]>([]);
+
+  const [documentationDescription, setDocumentationDescription] =
+    useState<string>('');
+  const [ongoingDescription, setOngoingDescription] = useState<string>('');
+  const [todoDescription, setTodoDescription] = useState<string>('');
+  const [doneDescription, setDoneDescription] = useState<string>('');
+
+  const [documentationEditing, setDocumentationEditing] =
+    useState<boolean>(false);
+  const [ongoingEditing, setOngoingEditing] = useState<boolean>(false);
+  const [todoEditing, setTodoEditing] = useState<boolean>(false);
+  const [doneEditing, setDoneEditing] = useState<boolean>(false);
+
+  const { id } = useParams();
+  const [userId, setUserId] = useState<string>('');
+
+  useEffect(() => {
+    const filterTaskByStatus = (status: string) => {
+      return allTasks.filter((task) => task.status === status);
+    };
+    const setUser = () => {
+      const auth = getAuth();
+      onAuthStateChanged(auth, async (user: User | null) => {
+        if (user) {
+          setUserId(user.uid);
+        }
+      });
+    };
+    setDocumentation(filterTaskByStatus('documentation'));
+    setOngoing(filterTaskByStatus('ongoing'));
+    setTodo(filterTaskByStatus('to_do'));
+    setDone(filterTaskByStatus('done'));
```

Comment:
Same as the comment I made elsewhere, make these values into constant and use them instead of raw string values.